projects/WTI-UI/src/app/modules/clarifications/components/new-clarification/new-clarification.component.ts

```diff
@@ -39,15 +39,21 @@ export class NewClarificationComponent implements OnInit, OnDestroy {
 
     this._teamService.postClarification(newClarification)
       .pipe(takeUntil(this._unsubscribe))
-      .subscribe(_ => {
-        this.newClarificationForm.reset();
-        this.close();
-        this._contestService.clarificationsUpdated.next();
-        this._uiHelper.alertOk('Clarification has been submitted successfully!');
-      }, (error: any) => {
-        console.error('error submitting new clarification');
-        console.error(error);
-      });
+      .subscribe({
+		next: value  => {
+	      this.newClarificationForm.reset();
+	      this.close();
+	      this._contestService.clarificationsUpdated.next();
+	      this._uiHelper.alertOk('Clarification has been submitted successfully! ');
+	    },
+	    error: (error: any) => {
+	      this._uiHelper.alertError('Error submitting clarification!');
+	      console.error('error submitting new clarification');
+	      console.error(error);
+	    },
+	    complete: () => {
```

Comment:
Is this empty routine necessary?

Reply:
Well this helps people looking at the code know that there is another callback option called complete.